lib/grell/capybara_driver.rb

```diff
@@ -13,7 +13,11 @@ def self.setup(options)
 
     def setup_capybara
       @poltergeist_driver = nil
-      Capybara.register_driver :poltergeist_crawler do |app|
+
+      # Capybara will not re-run the block if the driver name already exists, so the driver name
+      # will have a time integer appended to ensure uniqueness.
+      driver_name = "poltergeist_crawler_#{Time.now.to_i}".to_sym
```

Comment:
Sounds good. If I make the following change in crawler.rb `initialize`, we will have access to the logger:
```ruby
if options[:logger]
  Grell.logger = options[:logger]
else
  Grell.logger = Logger.new(STDOUT)
end

@driver = CapybaraDriver.setup(options)
```